### ISSUE TITLE
[question] Documentation for the using dam option?

### ISSUE THREAD

**Issue by KMarkert** [c-h-david/rapid#25]
Hi there,

I am looking to model a watershed which includes some reservoirs/dams. I see that there is something in the namelist regarding a dam file but have not found more information past that in the repo or on the [docs page](http://rapid-hub.org/documents.html)

I was wondering if you have an example file that I could go off of for my watershed or any documentation on how to prepare the model for use if there is a dam present?

Thanks in advance!

Kel

**Comment by c-h-david** [comment]
Hi Kel,
You probably have two options for including dams.  The first would be to replace dam outflows by in situ measurements as documented in [David et al. (2011, HP)](https://doi.org/10.1002/hyp.8070).  In the code, this is generally referred to as `forcing`.  You could also activate an equation for simulating dams that uses characteristic quantities associated with each one of the dams.  This is referred to as `dam` in the code, but is still highly experimental, i.e. _to be used with caution_. You might want to talk with Ahmad Tavakoly for example files.  Ahmad designed the math and a draft Fortran implementation, and I did the implementation within RAPID.  Hope this helps!
Cedric

**Comment by KMarkert** [comment]
Hi Cedric, Thanks for the response and apologies for my delayed response!
The project I am working in is looking at future climate projections and in regulated basins so using the `forcing` option may be difficult given we won't have observations or rules for the future.
The `dam` option seems more appropriate for this use case (if I am understanding things correctly). Unfortunately, I do not have Ahmad's contact information to reach out. Is there documentation or somewhere in the code where I can read through how to format things? Or do you have any direction on what kind of inputs are required?

**Comment by c-h-david** [comment]
Hi Kel, I tried to email Ahmad's contact to you, but I guess you might have a new address. Email me?

**Comment by aat669** [comment]
Hi Kel, We implemented a non-data driven reservoir routing in the RAPID model. We intended to have reservoir routing with minimum reservoir data.  The required reservoir data include minimum and maximum reservoir storage (in m3). I recently completed the final experiments and am half through a journal paper draft which will introduce RAPID reservoir routing along with performance evaluation, and an appendix to show how to use it.
